tests/utils/kudo.go

```diff
@@ -149,6 +149,19 @@ func (c *KubernetesTestClient) LogObjectsOfKinds(namespace string, components []
 	}
 }
 
+func (c *KubernetesTestClient) PrintLogsOfPod(podName, namespace string) {
+	kubectlPath := getKubectlPath()
+	log.Info(fmt.Sprintf("Using kubectl from path: %s", kubectlPath))
+
+	cmd := exec.Command(kubectlPath, "logs", podName, fmt.Sprintf("--namespace=%s", namespace))
+	log.Infoln(fmt.Sprintf("logs %s --namespace=%s", podName, namespace))
+	out, err := cmd.Output()
+	if err != nil {
+		log.Error(string(err.(*exec.ExitError).Stderr))
```

Comment:
I think this is unnecessary, casting the error to `ExitError` and getting `Stderr` and can leave out possible `errors` that aren't `ExitError` like if the file isn't an executable or file not found, which `exec.Run` wrap inside generic `Error` type and not `ExitError` type. 

Its ok to just to print the error as it comes, and return it